src/Funds/FundForm/FundForm.js

```diff
@@ -410,6 +414,16 @@ const FundForm = ({
                         </Col>
                       </Row>
                     </Accordion>
+                    <Accordion
+                      closedByDefault
+                      id={SECTIONS_FUND.DONOR_INFORMATION}
+                      label={<FormattedMessage id="ui-finance.fund.information.donorInformation" />}
+                    >
+                      <DonorsList
```

Comment:
"Add donor" btn lives in `<DonorsList>` components? It seems this point was missed during the review of the https://github.com/folio-org/stripes-acq-components/pull/724
As for me, those should be separate components.

Reply:
@usavkov-epam I am thinking of making it configurable through props.